### PR TITLE
Feature equipment usage details

### PR DESCRIPTION
**Pull request by yeoxhshelby**
Added icons, fixed conflicts, did some refactoring and removed codes that are not useful :)

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -1,6 +1,9 @@
 DB_USERNAME = postgres
 DB_PASSWORD= root
+<<<<<<< HEAD
 DB_HOST = localhost
+=======
+>>>>>>> 08190f34c260c3cc0249c291abc0ad327186ed7c
```
Comment (georgeyinxu) [inline backend/.env]: Please remove the head lines

File: frontend/src/pages/TodayProduction.js [hunk 2]
```diff
@@ -1,105 +1,122 @@
-import React, { useEffect, useState } from "react";
-import "../styles/tpd.css";
-import ProductChart from "../components/tpd/ProductChart";
-import TableComponent from "../components/tpd/TableComponent";
+import React, { useEffect, useState } from 'react';
+import '../styles/tpd.css';
+import ProductChart from '../components/tpd/ProductChart';
+import TableComponent from '../components/tpd/TableComponent';
+<<<<<<< HEAD
```
Comment (georgeyinxu) [inline frontend/src/pages/TodayProduction.js]: Remove the head here

File: frontend/src/pages/TodayProduction.js [hunk 3]
```diff
@@ -1,105 +1,122 @@
...
 // import Modal from '../components/tpd/Modal';
-import TextField from "@mui/material/TextField";
-import { Tab } from "@mui/material";
-import Sidebar from "../components/sidebar/Sidebar";
+import TextField from '@mui/material/TextField';
+import { Tab } from '@mui/material';
+import Sidebar from '../components/sidebar/Sidebar';
 // import { Link, DirectLink, Element, Events, animateScroll as scroll, scrollSpy, scroller } from 'react-scroll';
+=======
```
Comment (georgeyinxu) [inline frontend/src/pages/TodayProduction.js]: Remove this

Comment (georgeyinxu) [reply frontend/src/pages/TodayProduction.js]: Check your stuff in the future please

File: frontend/src/pages/TodayProduction.js [hunk 4]
```diff
@@ -1,105 +1,122 @@
...
+import {
+  Link,
+  DirectLink,
+  Element,
+  Events,
+  animateScroll as scroll,
+  scrollSpy,
+  scroller,
+} from 'react-scroll';
+>>>>>>> 08190f34c260c3cc0249c291abc0ad327186ed7c
```
Comment (georgeyinxu) [inline frontend/src/pages/TodayProduction.js]: Here too

File: backend/.env [hunk 5]
```diff
@@ -1,6 +1,9 @@
 DB_USERNAME = postgres
 DB_PASSWORD= root
+<<<<<<< HEAD
```
Comment (georgeyinxu) [inline backend/.env]: Please remove this (Error during merge)

File: backend/.env [hunk 6]
```diff
@@ -1,6 +1,9 @@
...
 DB_HOST = localhost
+=======
```
Comment (georgeyinxu) [inline backend/.env]: Please remove this (Error during merge)

File: backend/.env (hunk 1 above)
Comment (georgeyinxu) [inline backend/.env]: Please remove this (Error during merge)

**Review by georgeyinxu — COMMENTED** [review]
Please change accordingly @hehshelby

**Review by georgeyinxu — APPROVED** [review]
Looks good, thanks shelby

**Review by Louis03pgx — APPROVED** [review]
lgtm

**Pull request merged**